api/lib/domain/models/OrganizationRole.js

```diff
@@ -0,0 +1,10 @@
+class OrganizationRole {
```

Comment:
mettre une enum de tous les roles possibles, à référencer aux endroits où c'est utiliser

Reply:
To discuss

Reply:
Pour l'instant il n'y a qu'un seul rôle orga, qui n'est pas encore utilisé explicitement dans le code. Du coup avoir une enum avec les rôles possibles est un peu prématuré à mon goût, mais sera peut être nécessaire une fois qu'on aura vraiment besoin de les manipuler ...